xmppserver/src/main/webapp/server-session-details.jsp

```diff
@@ -318,12 +321,15 @@
                                 </c:if>
                                 <td nowrap>
                                     <c:choose>
-                                    <c:when test="${session.usingServerDialback}">
-                                        <fmt:message key="server.session.details.dialback"/>
-                                    </c:when>
-                                    <c:otherwise>
-                                        <fmt:message key="server.session.details.tlsauth"/>
-                                    </c:otherwise>
+                                        <c:when test="${session.usingServerDialback}">
+                                            <fmt:message key="server.session.details.dialback"/>
+                                        </c:when>
+                                        <c:when test="${session.usingSaslExternal}">
+                                            <fmt:message key="server.session.details.tlsauth"/>
```

Comment:
The name of key doesn't seem to reflect it's usage here. It may be necessary to rename the key?

Reply:
I don't see the problem? SASL EXTERNAL == TLS authentication (at least in the current implementation of Openfire).